willow/Dockerfile

```diff
@@ -1,17 +1,20 @@
 FROM ruby:2.3
 
+# Setup build variables
+ARG RAILS_ENV=production
+
 # Set up environmental variables - do all in one go to reduce cache layers (see https://docs.docker.com/engine/reference/builder/#env)
 # Default to UTF-8 file.encoding
 # see https://bugs.debian.org/775775
 # and https://github.com/docker-library/java/issues/19#issuecomment-70546872
-ENV LANG=C.UTF-8 \
+ENV RAILS_ENV="$RAILS_ENV" \
+    LANG=C.UTF-8 \
     JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/jre \
     RAILS_LOG_TO_STDOUT=yes_please \
     PATH=/fits/fits-1.0.2/:$PATH \
-    APP_HOME=/willow \
-    BUNDLE_GEMFILE=/willow/Gemfile \
     BUNDLE_JOBS=2 \
-    BUNDLE_PATH=/bundle
+    APP_PRODUCTION=/willow.production/ \
```

Comment:
Interesting choice. Any particular reason for using a separate directory for dev vs production? In theory they would be different images, so there shouldn't be any clashing. Just curious to what led to using a different dir.